plaso/cli/image_export_tool.py

```diff
@@ -296,8 +299,12 @@ def _ExtractWithFilter(
       source_path_specs (list[dfvfs.PathSpec]): path specifications to extract.
       destination_path (str): path where the extracted files should be stored.
       output_writer (CLIOutputWriter): output writer.
+      artifacts_registry (ArtifactRegistry): Artifacts registry object.
+      artifact_filters_path (str): path of the file that contains the
+          names of the artifacts filter definitions or definitions directly
```

Comment:
Let's not overload paths here, this makes the code hard to follow and more error prone.

Split artifact_filters_path and artifact_filters